nym-vpnc/Cargo.toml

```diff
@@ -11,13 +11,14 @@ license.workspace = true
 # See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
 
 [dependencies]
-clap.workspace = true
+clap = { workspace = true, features = ["derive"] }
 prost.workspace = true
-tokio.workspace = true
+tokio = { workspace = true, features = ["full"]}
```

Comment:
did it not work without `full`? Note that adding more features can increase compile time even if the components are not used